src/crawlers/playwright_crawler.js

```diff
@@ -91,9 +95,9 @@ import { apifyOptionsToLaunchOptions, getPlaywrightLauncherOrThrow } from '../pl
  *     };
  * ]
  * ```
-* @property {object} [launchContext]
- *   Options used by {@link Apify#launchPlaywright} to start new Playwright instances.
- *  * @property {number} [handlePageTimeoutSecs=60]
+ * @property {LaunchPlaywrightOptions} [launchContext]
```

Comment:
I did not think of this before, but it looks a bit confusing. Options -> Context. And I'm also thinking about future TS types. I think we should have `LaunchContext` and then`PuppeteerLaunchContext` and `PlaywrightLaunchContext` which would both extend the original `LaunchContext` and the original could even be required from `browser-pool` itself. What do you think?